src/hip_hcc.cpp

```diff
@@ -782,10 +782,12 @@ hipError_t ihipDevice_t::initProperties(hipDeviceProp_t* prop) {
     DeviceErrorCheck(err);
 
     // BDFID is 16bit uint: [8bit - BusID | 5bit - Device ID | 3bit - Function/DomainID]
```

Comment:
Are the last 3bits of BDFID still Function/Domain? Aren't BusID and DeviceID inquired by a separate HSA API call too?